nemo/collections/asr/parts/submodules/multi_head_attention.py

```diff
@@ -362,3 +609,52 @@ def forward(self, x, cache_len=0):
         if self.dropout_emb:
             pos_emb = self.dropout_emb(pos_emb)
         return self.dropout(x), pos_emb
+
+
+class ChunkedRelPositionalEncoding(PositionalEncoding):
```

Comment:
Would you please add a reference to the paper and some description on what it does?

Reply:
If this class inherits from the relative positional encoding class, can you drop some of the methods here  if they are the same as the upper class?

Reply:
The methods aren't the same

Reply:
Let's give a hint or ref to the longformer and mention it can be used with which self-attention class.